bundles/CoreBundle/src/Migrations/Version20230320110429.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230320110429 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'rename default dir for symfony-config files';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->renameConfigFolder(['image-thumbnails', 'video-thumbnails'], '-', '_');
```

Comment:
We should mention this change in upgrade notes as well, just in case someone has this configs in VCS.